src/BuiltInTools/dotnet-watch/Program.cs

```diff
@@ -67,7 +67,13 @@ public Program(IConsole console, string workingDirectory)
             // AppContext.BaseDirectory = $sdkRoot\$sdkVersion\DotnetTools\dotnet-watch\$version\tools\net6.0\any\
             // MSBuild.dll is located at $sdkRoot\$sdkVersion\MSBuild.dll
             var sdkRootDirectory = Path.Combine(AppContext.BaseDirectory, "..", "..", "..", "..", "..", "..");
+#if DEBUG
+            // In the usual case, use the SDK that contains the dotnet-watch. However during local testing, it's
+            // much more common to run dotnet-watch from a different SDK. Use the ambient SDK in that case.
+            MSBuildLocator.RegisterDefaults();
```

Comment:
Annoying local build issue where it would use the dogfood SDK. This was causing the annoying FailFast error that I poked you about.